docs/usage/configuration-options.md

```diff
@@ -753,7 +753,7 @@ image: my.old.registry/aRepository/andImage:1.18-alpine
   "customManagers": [
     {
       "customType": "regex",
-      "fileMatch": ["values.yaml$"],
+      "filePatterns": ["**values.yaml"],
```

Comment:
```suggestion
      "filePatterns": ["**/values.yaml"],
```
I think it needs the slash 

https://github.com/renovatebot/renovate/pull/34615#discussion_r1983351635